app/src/main/java/io/github/joshkergan/giftr/people/PeopleAdapter.java

```diff
@@ -54,7 +54,7 @@ public PeopleAdapter(SQLiteDatabase giftrDb) {
     public void onBindViewHolder(ViewHolder holder, int position) {
         c.moveToPosition(position);
         byte[] blob = c.getBlob(c.getColumnIndex(PeopleContract.PeopleEntry.COLUMN_NAME_PHOTO));
-        holder.mFriendIcon.setImageBitmap(BitmapFactory.decodeByteArray(blob, 0, blob.length));
+        //holder.mFriendIcon.setImageBitmap(BitmapFactory.decodeByteArray(blob, 0, blob.length));
```

Comment:
Was this the line that you said was failing? If it was it's because we lacked a null check on the data set.